File: Support/Fuego/Pythia/pythia-0.4/packages/journal/lib/Console.cc
// -*- C++ -*-
//
//--------------------------------------------------------------------------------
//
//                              Michael A.G. Aivazis
//                       California Institute of Technology
//                       (C) 1998-2003  All Rights Reserved
//
// <LicenseText>
//
//--------------------------------------------------------------------------------
//

#include <portinfo>
#include <ostream>
#include <string>

#include "Device.h"
#include "StreamDevice.h"
#include "Console.h"

#include <iostream>

using namespace journal;

// meta-methods
journal::Console::Console() :
    StreamDevice(std::cout)
{}

Console::~Console() {}

// version
// $Id$

// End of file

File: Support/Fuego/Pythia/pythia-0.4/packages/pyre/pyre/journal/Console.py
#!/usr/bin/env python
#
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#
#                             Michael A.G. Aivazis
#                      California Institute of Technology
#                      (C) 1998-2003  All Rights Reserved
#
# <LicenseText>
#
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#


from Device import Device


class Console(Device):


    def createDevice(self):
        from journal.Console import Console
        return Console()


    def __init__(self):
        Device.__init__(self, "console")
        return


# version
__id__ = "$Id$"

# End of file 
